### PR TITLE
Fix redirectTo returned by `useEditController`

### PR DESCRIPTION
**Pull request by WiXSL**
`useEditController` should return the redirectTo used to redirect (if any) or the default, just like `useCreateController` does it, not the default

### CODE REVIEW

**Comment by fzaninotto** [comment]
I'm not sure what bug this fixes. Could you please add a unit test?

**Review by slax57 — APPROVED** [review]

**Comment by slax57** [comment]
Seems good to me! I'll just wait on François' approval before merging

File: packages/ra-core/src/controller/edit/useEditController.ts
Comment (fzaninotto) [inline packages/ra-core/src/controller/edit/useEditController.ts:223]: This context variable isn't used anywhere in our code - I think it's a leftover from v3. I'll validate the PR, but I think we should ultimately mark it as deprecated.

**Review by fzaninotto — APPROVED** [review]

**Pull request merged**
